Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+EndUser.swift

```diff
@@ -33,20 +33,8 @@ public struct EndUserAttributes: SpanAttributeNamespace {
     public init(attributes: SpanAttributes) {
         self.attributes = attributes
     }
-
-    public subscript<T>(dynamicMember member: KeyPath<NestedAttributes, SpanAttributeKey<T>>) -> SpanAttribute? {
-        get {
-            let key = NestedAttributes.namespace[keyPath: member]
-            return self.attributes[key.name]
-        }
-        set {
-            let key = NestedAttributes.namespace[keyPath: member]
-            self.attributes[key.name] = newValue
-        }
-    }
-
-    public enum NestedAttributes: NestedSpanAttributesProtocol {
-        case namespace
```

Comment:
we can't get away with the enum if we want to implement the subscripts generically... because the enums and protocols thingy being only 5.3+, so we require an init.